src/util/helpers/pseudonymRegistration.js

```diff
@@ -0,0 +1,63 @@
+import api from '@molgenis/molgenis-api-client'
+
+/**
+ * Helper function for deciding if this field need to be a PseudonymRegistrationComponent
+ *
+ * @param field
+ * @returns {Function} Function which evaluates to a boolean
+ */
+const isPseudonymRegistrationComponent = (field) => {
+  return field.hasOwnProperty('tags') && field.tags.length > 0 && field.tags.findIndex(tag => tag.objectIRI === 'http://purl.obolibrary.org/obo/NCIT_C142654') !== -1
+}
+
+const requestConfiguration = (id) => api.get(`/api/v2/PseudonymRegistrationConfig?q=ID=like=${id}`)
+
+const submitPseudonymRegistration = async (config, originalID) => {
+  let requestID = null
+  const postOptions = { body: JSON.stringify({ OriginalID: originalID }) }
+  try {
+    await api.post(`/api/data/${config.LinkEntityName}`, postOptions).then(async response => {
+      if (response.status === 201) {
+        requestID = await getNewPseudonym(config, originalID)
+      } else {
+        throw new Error(`Unexpected status code ${response.status}`)
+      }
+    }, async error => {
+      if (error.status === 400) {
+        await checkForDuplicateID(config, originalID)
+      } else {
+        throw new Error(`${error.statusText} Please contact a system administrator`)
+      }
+    })
+  } catch (error) {
+    await Promise.reject(error.toString())
+  }
+  return requestID
+}
+
+export default {
+  isPseudonymRegistrationComponent, requestConfiguration, submitPseudonymRegistration
+}
+
```

Comment:
Would be nice to have the export at the top of the file so you can immediately see what is exposed.